keys/keys.go

```diff
@@ -12,28 +12,32 @@ import (
 	"strings"
 
 	"github.com/btcsuite/btcd/btcec"
-	"github.com/cosmos/go-bip39"
-
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/go-bip39"
+	"github.com/kava-labs/go-sdk/tx"
+	"github.com/kava-labs/kava/app"
 	"github.com/tendermint/tendermint/crypto"
 	"github.com/tendermint/tendermint/crypto/secp256k1"
 )
 
 const (
 	defaultBIP39Passphrase = ""
-	// TODO: Change BIP44Prefix
-	BIP44Prefix = "44'/714'/"
-	PartialPath = "0'/0/0"
-	FullPath    = BIP44Prefix + PartialPath
+	BIP44Prefix            = "44'/188'/"
```

Comment:
Should be `118` not `188` (explains test output)

Reply:
https://github.com/Kava-Labs/go-sdk/pull/1/commits/64d1867e55f9a57b0d3c1c7b21d937ee0b874d62